src/yb/master/catalog_manager.cc

```diff
@@ -5919,7 +5928,11 @@ void CatalogManager::ReportMetrics() {
   // Report metrics on how many tservers are alive.
   TSDescriptorVector ts_descs;
   master_->ts_manager()->GetAllLiveDescriptors(&ts_descs);
-  metric_num_tablet_servers_live_->set_value(ts_descs.size());
+  const int32 num_live_servers = ts_descs.size();
+  metric_num_tablet_servers_live_->set_value(num_live_servers);
+  
```

Comment:
nit: can you pls get rid of the white space on this line?

> Lint: Trailing Whitespace
> This line contains trailing whitespace. Consider setting up your editor to automatically remove trailing whitespace, you will save time.
> 
> Lint: whitespace/end_of_line
> Line ends in whitespace. Consider deleting these extra spaces.
>

Reply:
Done.